app/assets/stylesheets/dashboard.scss

```diff
@@ -1,9 +1,9 @@
 @import 'variables';
 @import 'mixins';
 .dashboard-container {
-  // width: 800px;
-  // max-width: 96%;
-  // margin: auto;
+  width: 800px;
+  max-width: 96%;
+  margin: auto;
```

Comment:
Hey, we intentionally commented out that code due to some other work that is currently being done around dashboard.

If you don't mind, I'd suggest different solution to this issue: [in the same file, around line #672](https://github.com/thepracticaldev/dev.to/blob/8738c4cf41f9e88d59dbf6e43d13cb8a72a2098f/app/assets/stylesheets/dashboard.scss#L672) let's add `margin: 0 auto;`. This should fix the layout and it would also not overlap with the work we're currently doing. Do you think you could make that change?

Reply:
@ludwiczakpawel, will make the changes. Thanks for the feedback.